### ISSUE TITLE
DRCSim 3.1.1 update via apt no longer loads robot models

### ISSUE THREAD

**Issue by osrf-migration** [osrf/drcsim#407]
**Original report ([archived issue](https://osrf-migration.github.io/drcsim-gh-pages/#!/osrf/drcsim/issues/407)) by Patrick Beeson (Bitbucket: [Patrick Beeson](https://bitbucket.org/%7Bd33f061a-d3c9-4b15-96cc-d08b463d97c6%7D/)).**

----------------------------------------
I think maybe the /usr/share/drcsim/setup.sh file wasn't properly changed.  When running 'roslaunch drcsim_gazebo atlas.launch', it can't find the models in atlas_description

Error [ModelDatabase.cc:407] Unable to download model[model://multisense_sl_description/meshes/head.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/r_uleg.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/l_foot.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/l_lleg.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/l_uleg.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/r_hand.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/r_farm.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/r_larm.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/r_uarm.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/r_scap.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/r_clav.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/r_lleg.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/r_foot.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/pelvis.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/ltorso.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/mtorso.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/utorso.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/l_clav.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/l_scap.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/l_uarm.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/l_larm.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/l_farm.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/l_hand.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/r_talus.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/l_talus.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/l_lglut.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/l_uglut.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/r_uglut.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/r_lglut.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://multisense_sl_description/meshes/head_camera.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://multisense_sl_description/meshes/head.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/pelvis.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/ltorso.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/mtorso.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/utorso.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/l_clav.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/l_scap.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/l_uarm.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/l_larm.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/l_farm.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/l_hand.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/r_clav.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/r_scap.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/r_uarm.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/r_larm.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/r_farm.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/r_hand.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/l_uleg.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/l_lleg.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/l_foot.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/r_uleg.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/r_lleg.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/r_foot.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/l_uglut.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/l_lglut.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/l_talus.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/r_uglut.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/r_lglut.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://atlas_description/meshes/r_talus.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified
Error [ModelDatabase.cc:407] Unable to download model[model://multisense_sl_description/meshes/head_camera.dae]
Error [SystemPaths.cc:371] File or path does not exist[""]
Error [Visual.cc:2137] No mesh specified

**Comment by osrf-migration** [comment]
**Original comment by Patrick Beeson (Bitbucket: [Patrick Beeson](https://bitbucket.org/%7Bd33f061a-d3c9-4b15-96cc-d08b463d97c6%7D/)).**

----------------------------------------
Adding /opt/ros/groovy/share to the GAZEBO_MODEL_PATH fixed things.  This should have been done by drcsim setup.sh methinks.

**Comment by osrf-migration** [comment]
**Original comment by Jose Luis Rivero (Bitbucket: [Jose Luis Rivero](https://bitbucket.org/%7Bd12309b2-f745-42ee-b119-aec4fcdf81fe%7D/), GitHub: [j-rivero](https://github.com/j-rivero)).**

----------------------------------------

* set assignee_account_id to "557058:155a32e2-420c-4d50-98e0-0e722f63f906"
* set assignee to "_jrivero_ (Bitbucket: [_jrivero_](https://bitbucket.org/_jrivero_), GitHub: [j-rivero](https://github.com/j-rivero))"

Sorry my bad. I'm releasing drcsim-3.1.1-2 which should fix the problem.

Meanwhile, as a workaround could you try:
```
cd /opt/ros/$ROS_DISTRO/share/drcsim_model_resources/gazebo_models/environments/
sudo ln -s ../../../atlas_description .
sudo ln -s ../../../irobot_hand_description .
sudo ln -s ../../../multisense_sl_description .
```
or even
```
export GAZEBO_MODEL_PATH=$GAZEBO_MODEL_PATH:/opt/ros/${ROS_DISTRO}/share/
```
should work.

**Comment by osrf-migration** [comment]
**Original comment by Jose Luis Rivero (Bitbucket: [Jose Luis Rivero](https://bitbucket.org/%7Bd12309b2-f745-42ee-b119-aec4fcdf81fe%7D/), GitHub: [j-rivero](https://github.com/j-rivero)).**

----------------------------------------

* changed state from "new" to "resolved"

It should be fixed now, with version 3.1.1-2.

Please reopen if needed. Thanks @pbeeson for your quick report and workaround.
